molr-mole-core/src/main/java/io/molr/mole/core/single/SingleNodeMissionExecutor.java

```diff
@@ -26,8 +27,9 @@
 
 public class SingleNodeMissionExecutor<R> implements MissionExecutor {
```

Comment:
What happens with the other moles (not only Single node missions)? (E.g.  I would have suggested that they would get automatic logging if the follow slf4j conventions ...  (e.g. RunnableLeafsMole ... or others to come ..). Is this not the case?